lang/test/org/partiql/lang/eval/visitors/GroupByItemAliasVisitorTransformTests.kt

```diff
@@ -0,0 +1,210 @@
+package org.partiql.lang.eval.visitors
+
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ArgumentsSource
+import org.partiql.lang.util.ArgumentsProviderBase
+
+class GroupByItemAliasVisitorTransformTests : VisitorTransformTestBase() {
```

Comment:
These are new tests, largely based off the tests from #298.

Reply:
Great!